src/lib/dnssd/TxtFields.cpp

```diff
@@ -161,6 +161,13 @@ uint8_t GetCommissioningMode(const ByteSpan & value)
     return MakeU8FromAsciiDecimal(value);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+uint8_t GetJointFabricMode(const ByteSpan & value)
```

Comment:
Probably return `BitFlags<JointFabricMode>` here too?  Like:

```
    return BitFlags<JointFabricMode>(MakeU8FromAsciiDecimal(value));
```

or so.

Reply:
Addressed in the latest commit. Thanks for your feedback.